src/bundle/View/EzView.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUiBundle\View;
+
+use EzSystems\EzPlatformAdminUiBundle\View\Template\EzTemplate;
+use Pagerfanta\View\DefaultView;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class EzView extends DefaultView
```

Comment:
Naming... AFAICS this is tied strictly to Pagerfanta. Don't really know how this works, but `EzView` is too generic. Maybe `EzPagerfantaView`?
While at it, maybe add some PhpDoc on what it is and what it does, so noob like me understands it :)